plugins/filters/add_prefix.py

```diff
@@ -0,0 +1,17 @@
+from ansible import utils
+
+
+def add_prefix(value, prefix=''):
+    """Add string prefix to item.
+
+    {{ ['hanukkah', 'easter', 'pessah'] | map('add_prefix', 'happy ' }}
+    -> ['happy hanukkah', 'happy easter', 'happy pessah']
+    """
+
+    return prefix + value
```

Comment:
OK, I read Guido's essay and looks like join is the best way. will change it. thanks.